transcom/mymove
# B-21487 UB Allowance in GetMoveTaskOrder response - INT

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21487)

## Summary

We needed to add the UB allowance to the response for the prime getMoveTaskOrder endpoints.

AC:
As the GHC prime,
I need to see the customer's baggage allowance on the move task order,
So that I can properly identify excess cost (or counsel the customer)

Given a task order called from get MoveTaskOrder,
When I review the entitlements section,
Then I can see a value for unaccompaniedBaggageAllowance (example:
"unaccompaniedBaggageAllowance": 2000,)

### How to test

1. Call the prime v1, v2, and v3 getMoveTaskOrder endpoints, and verify that the unaccompaniedBaggageAllowance is now returned in the response.

V1:
![image](https://github.com/user-attachments/assets/196d05ee-1db0-441a-a097-b7f27b1c22f2)
![image](https://github.com/user-attachments/assets/357ff28b-83f2-4bb9-bbae-f232ec200041)

V2:
![image](https://github.com/user-attachments/assets/a4c87c03-96ee-455b-9f6d-1accb3bfc435)
![image](https://github.com/user-attachments/assets/150fc099-eeac-4024-8387-dd9515ac2ed1)

V3:
![image](https://github.com/user-attachments/assets/fe530650-8fc2-4036-8895-fc48decbcf60)
![image](https://github.com/user-attachments/assets/edc6cb44-de86-4178-9650-e32ff076f6b0)
